### PR TITLE
Book implement

### PR DESCRIPTION
**Pull request by volodymyr-komarnyckyi**

### CODE REVIEW

**Review by YKonoplyov — APPROVED** [review]

File: book/models.py [hunk 1]
```diff
@@ -20,6 +32,7 @@ class Book(models.Model):
     )
     inventory = models.PositiveIntegerField(default=0)
     daily_fee = models.DecimalField(max_digits=6, decimal_places=2)
+    image = models.ImageField(null=True, upload_to=movie_image_file_path)
```
Comment (anonymousprojectuser) [inline book/models.py:35]: replace movie to book

File: book/models.py [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
     HARD = "Hard cover"
     SOFT = "Soft cover"
 
 
+def movie_image_file_path(instance, filename):
+    _, extension = os.path.splitext(filename)
+
+    filename = f"{slugify(instance.title)}-{uuid.uuid4()}.{extension}"
+
+    return os.path.join("uploads/movies/", filename)
```
Comment (anonymousprojectuser) [inline book/models.py:20]: replace movie to book

File: book/models.py [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
 from django.db import models
+from django.utils.text import slugify
 
 
 class CoverType(Enum):
     HARD = "Hard cover"
     SOFT = "Soft cover"
 
 
+def movie_image_file_path(instance, filename):
```
Comment (anonymousprojectuser) [inline book/models.py:15]: replace movie to book

File: book/views.py [hunk 4]
```diff
@@ -1,13 +1,36 @@
...
+    )
+    def upload_image(self, request, pk=None):
+        play = self.get_object()
+        serializer = self.get_serializer(play, data=request.data)
+
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_200_OK)
+
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```
Comment (oleksiikolii) [inline book/views.py:36]: No blank line(

**Review by oleksiikolii — APPROVED** [review]

**Pull request merged**